### ISSUE TITLE
Sidekiq as service doesnt execute system commands

### ISSUE THREAD

**Issue by lautarocastillo** [mperham/sidekiq#4092]
Hi guys, I'm having this issue:
when I run sidekiq like this:  `bundle exec sidekiq` it works perfectly. But when I run like this: `systemctl enable sidekiq` (following the official guide), it works everything, BUT... not the system commands.
I only want to do this: ```
    pid = system('mpg123','-q', file)
```
What could be the problem?

**Comment by lautarocastillo** [comment]
Thanks for your answer Mike!
I followed many guides, but I can't make it work. Isn't  adding this line enough `Environment=RAILS_ENV=production` ?

**Comment by mperham** [comment]
No, that's not enough.  That's not a full user login environment with a custom PATH.  If you are dependent on user-specific installed software in /home/someuser/bin for instance, you need to do something like:

```
User=someuser
ExecStart=/usr/bin/bash -l -c "bundle exec sidekiq"
```

I'd suggest you read up on systemd, bash login and environment.
